packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx

```diff
@@ -0,0 +1,146 @@
+"use client";
+
+import * as React from "react";
+import cx from "clsx";
+
+import ErrorFormTooltip from "../../ErrorFormTooltip";
+import FormLabel from "../../FormLabel";
+import { SIZE_OPTIONS, RESIZE_OPTIONS } from "./consts";
+import useErrorTooltip from "../../ErrorFormTooltip/hooks/useErrorTooltip";
+import getFieldDataState from "../../common/getFieldDataState";
+import type { Props } from "./types";
+import useRandomId from "../../hooks/useRandomId";
+import { getSpaceAfterClasses } from "../../common/tailwind";
+
+const Textarea = React.forwardRef<HTMLTextAreaElement, Props>((props, ref) => {
+  const {
+    size = SIZE_OPTIONS.NORMAL,
+    disabled,
+    resize = RESIZE_OPTIONS.VERTICAL,
+    dataTest,
+    id,
+    spaceAfter,
+    fullHeight,
+    defaultValue,
+    value,
+    label,
+    name,
+    error,
+    placeholder,
+    helpClosable = true,
+    maxLength,
+    onChange,
+    onFocus,
+    onBlur,
+    tabIndex,
+    help,
+    rows,
+    readOnly,
+    required,
+    dataAttrs,
+  }: Props = props;
+
+  const forID = useRandomId();
+  const inputId = id || forID;
+
+  const hasTooltip = Boolean(error || help);
+
+  const {
+    tooltipShown,
+    tooltipShownHover,
+    setTooltipShown,
+    setTooltipShownHover,
+    labelRef,
+    iconRef,
+    handleFocus,
+  } = useErrorTooltip({ onFocus, hasTooltip });
+
+  const shown = tooltipShown || tooltipShownHover;
+
+  return (
+    <label
+      className={cx(
+        "orbit-textarea",
+        "font-base relative flex w-full flex-col",
+        fullHeight && "h-full flex-1",
+        spaceAfter && getSpaceAfterClasses(spaceAfter),
+      )}
+      ref={labelRef}
+    >
+      {label && (
+        <FormLabel
+          filled={!!value}
+          error={!!error}
+          help={!!help}
+          required={required}
+          disabled={disabled}
+          iconRef={iconRef}
+          onMouseEnter={() => setTooltipShownHover(true)}
+          onMouseLeave={() => setTooltipShownHover(false)}
+        >
+          {label}
+        </FormLabel>
+      )}
+
+      <textarea
+        className={cx(
+          "w-full appearance-none",
+          "font-base",
+          "rounded-normal relative box-border block overflow-auto",
+          "min-h-form-box-normal",
+          "duration-fast transition-shadow ease-in-out",
+          "border border-transparent",
+          size === SIZE_OPTIONS.SMALL && "leading-small text-small px-sm py-xs",
+          size === SIZE_OPTIONS.NORMAL && "text-normal p-sm leading-normal",
+          error
+            ? "shadow-form-element-error hover:shadow-form-element-error"
```

Comment:
It should not be the same shadow on hover and on normal state

Reply:
good, solved in #4047, will add the class for now, but will work after the merge of Radio